src/cf-icons/src/cf-icons.less

```diff
@@ -44,24 +44,24 @@
 //
 // For example you can't do this:
 // .link-with-auto-icon {
-//   .@{cf-icon-prefix}();
+//   .@{cf-icon-prefix}(  );
 // }
 //
 // But you can do this:
 // .link-with-auto-icon {
-//   .@cf-icon();
+//   .@cf-icon(  );
 // }
 .cf-icon,
 .@{cf-icon-prefix} {
-  font-family: 'CFPB Minicons';
-  display: inline-block;
-  font-style: normal;
-  font-weight: normal;
-  line-height: 1;
-  // TODO: Determine whether this can be safely removed and remove it.
-  //       Font smooth/smoothing is non-standard:
-  //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
-  -webkit-font-smoothing: antialiased;
+    font-family: 'CFPB Minicons';
+    display: inline-block;
+    font-style: normal;
+    font-weight: normal;
+    line-height: 1;
+    // TODO: Determine whether this can be safely removed and remove it.
+    //       Font smooth/smoothing is non-standard:
+    //       https://developer.mozilla.org/en-US/docs/Web/CSS/font-smooth
```

Comment:
I had checked on both retina and non-retina (Thunderbolt Display and personal machines) but hadn't noticed the small type in the YouTube icon. Every other icon is crisper at the default setting, but it's probably not worth investigating more at this point. Instead I'm going to remove the `TODO` comment so no one else goes down this rabbit hole.